### DISCUSSION TITLE
The case for multiple front ends

### DISCUSSION THREAD

**Discussion by elipsion** [grindsa/acme2certifier#126] [Ideas]
As the back end library grows with ever more pluggable back ends, I think it's becoming more desirable to also have the fronts pluggable.

My particular use case involves a client software that only supports the MS `certsrv` "protocol", but I could also see SCEP being fairly straight forward to implement.

**Comment by elipsion** [comment]
Ping, @grindsa

**Comment by grindsa** [comment]
hi, sorry for letting you waiting. The idea of having additional protocol support sounds really interesting but I fear that I do not have the bandwidth to work on it. 

However patches are welcome at any time...
